qtconsole/qt_loaders.py

```diff
@@ -240,30 +240,37 @@ def import_pyside():
 
 def load_qt(api_options):
     """
-    Attempt to import Qt, given a preference list
-    of permissible bindings
+    Import Qt, given a preference list of permissible bindings.
 
-    It is safe to call this function multiple times.
+    It is safe to call this function multiple times.  If a Qt binding
+    has already been loaded, it will be returned regardless of the
+    preference list.
 
     Parameters
     ----------
     api_options: List of strings
-        The order of APIs to try. Valid items are 'pyside',
-        'pyqt', 'pyqt5', 'pyqtv1' and 'pyqtdefault'
+        The order of APIs to try. Valid items are 'pyside', 'pyqt',
+        'pyqt5', 'pyqtv1' and 'pyqtdefault'
 
     Returns
     -------
 
     A tuple of QtCore, QtGui, QtSvg, QT_API
-    The first three are the Qt modules. The last is the
-    string indicating which module was loaded.
+    The first three are the Qt modules. The last is the string
+    indicating which module was loaded.
 
     Raises
     ------
-    ImportError, if it isn't possible to import any requested
-    bindings (either becaues they aren't installed, or because
-    an incompatible library has already been installed)
+    ImportError, if it isn't possible to import any requested bindings
+    (because they aren't installed).
     """
+    if "PySide" in sys.modules:
```

Comment:
This should use the `loaded_api()` check that already exists in the module. It's possible, if unusual, to load the top-level namespace of one of these packages without actually loading the bindings:

```
In [1]: import PyQt4

In [2]: import PyQt5.QtCore

In [3]: 
```

`loaded_api()` checks whether `QtCore` is loaded.